crates/cdk-mintd/src/config.rs

```diff
@@ -104,14 +105,21 @@ pub struct FakeWallet {
     pub supported_units: Vec<CurrencyUnit>,
     pub fee_percent: f32,
     pub reserve_fee_min: Amount,
+    pub min_delay_time: u64,
+    pub max_delay_time: u64,
 }
 
 impl Default for FakeWallet {
     fn default() -> Self {
+        let mut rng = rand::thread_rng();
+        let min_rang = rng.gen_range(1..=1000);
+        let max_rang = rng.gen_range(1..=1000);
```

Comment:
I think 1000 seconds is too high for a default. This also doesnt check if the min is higher then the max. 

I would just set the min default to 1 and max to 3

Reply:
but setting the min at 1 and max at 3 is small, we can do 1 and 10

Reply:
For testing on a fake wallet waiting 10 seconds is a long time. Its a default if someone wants a it higher they can set it in the config